### PR TITLE
feat(addon-logs-archives): add logs archives for addons

### LINKED ISSUE #518: [Logs archive] Make it available for addons
The  `logs` command has a `--addon` flag. But the `logs-archives` command does not whereas they exist.

The request has the form `GET https://db-api-osc-fr1.scalingo.com/api/databases/${DB_ID}/logs_archives?cursor=`

While you are at it, maybe also fix this issue: https://github.com/Scalingo/developers-documentation/issues/147

### PR DESCRIPTION
**Pull request by sihamais**
Waiting for #701 to be merged

This PR also contains a couple of files renaming and functions renaming. You can filter out these commits to better see what this PR contains.

- [x] Add a changelog entry in the section "To Be Released" of CHANGELOG.md
- [x] Test it when AddonLogsArchives is released on [go-scalingo#235](https://github.com/Scalingo/go-scalingo/pull/235)

Fix #518

### CODE REVIEW

**Comment by sihamais** [comment]
1. I made a first draft for this feature but I have no way of testing it since I don't have any addon archive on my project. For now it tells me well that I have no archive. But I would like to test it more. Is there a way ? 
2. What's the difference between the `page` and the `cursor` flag ? And there is a function to get app logs archives with cursor flag but it isn't called anywhere (I think). So where is it used ?
@EtienneM

**Comment by EtienneM** [comment]
1. You should have an access to a staging app with database logs archives available. Is it OK?
2. I don't understand where you see this `cursor` thing. Can you clarify? About the logs archives for app with a `cursor` flag, can you point me to a file?

**Comment by sihamais** [comment]
1. Yep, it's good.
2. I've seen it in the request URL on the issue, and i've checked that there is a cursor param on the request made on the dashboard. Also, since app logs archives use it I figured addons logs archives would use it too.

**Comment by sihamais** [comment]
Actually, the function I've seen is in another project https://github.com/Scalingo/go-scalingo/blob/master/logs-archives.go#L33
(I got lost) Still don't know when `cursor` is used ? Should I just ignore it and use the `page` argument ?

**Comment by EtienneM** [comment]
The `cursor` is the _old_ way we used to use for the pagination. The new way is the `page` parameter, so please use this one. 

`cursor` was used by the old dashboard. Now that the old dashboard is deprecated, we will remove the support for `cursor`. Thanks to your report, I opened an issue on our side.

Note that [go-scalingo](https://github.com/Scalingo/go-scalingo) is the Go client to interact with our API.

**Comment by sihamais** [comment]
Okay, so go-scalingo is the Go client to interact with the API but it does not have a method to get the logs archives of the addon (here https://github.com/Scalingo/go-scalingo/blob/master/logs-archives.go). I actually implemented it yesterday but I just realized that i was editing on the dependancy (got lost). What should I do ? Normally it should be on the go-scalingo project but I'm not sure I have the right to open a PR there and even if I do I would have to wait for the next release to use it in the CLI. I'm kinda stuck here.

**Comment by EtienneM** [comment]
The process is indeed  to develop this feature on go-scalingo and open a PR there. I'm the one in charge of the go-scalingo release, so I'll proceed with a new release as soon as I merge your PR. That's what we usually do. You can see that in the changelog, many versions contain a single entry (https://github.com/Scalingo/go-scalingo/blob/master/CHANGELOG.md).

File: db/logs_archive.go
Comment (curzolapierre) [inline db/logs_archive.go:42]: Do you have an example of the output ?

**Review by curzolapierre — APPROVED** [review]
SGTM

File: db/logs_archive.go
Comment (EtienneM) [reply db/logs_archive.go:42]:
```╰─$ go build -o scalingo-cli ./scalingo && ./scalingo-cli -a scalingo-api-production logs-archives --addon addon_id                                                                 
To:   Fri Nov 5 13:10:56 +0000 UTC 2021
From: Fri Nov 5 09:15:53 +0000 UTC 2021
Size: 3705928                          
Url:  https://logs-archive-service-osc-fr1.scalingo.com/logs-archives/[REDACTED]/download/[REDACTED].log-2021110513.gz?sign=[REDACTED]&exp=1636131525               
-------                                                                                                                                                                                                            To:   Fri Nov 5 09:15:51 +0000 UTC 2021
From: Fri Nov 5 05:15:12 +0000 UTC 2021                                                                                                                                                                            
Size: 3551531                                                                                                                                                                                                      
Url:  https://logs-archive-service-osc-fr1.scalingo.com/logs-archives/[REDACTED]/download/[REDACTED].log-2021110509.gz?sign=[REDACTED]&exp=1636131525
-------
To:   Fri Nov 5 05:15:12 +0000 UTC 2021
From: Fri Nov 5 01:15:56 +0000 UTC 2021
Size: 3436225
Url:  https://logs-archive-service-osc-fr1.scalingo.com/logs-archives/[REDACTED]/download/[REDACTED].log-2021110505.gz?sign=[REDACTED]&exp=1636131525
```

**Pull request merged**
